modules/web/client/src/main/scala/observe/ui/Icons.scala

```diff
@@ -165,6 +180,9 @@ object Icons:
   inline def CircleSolid       = FontAwesomeIcon(faCircleSolid)
   inline def Clock             = FontAwesomeIcon(faClock)
   inline def Crosshairs        = FontAwesomeIcon(faCrosshairs)
+  inline def FileArrowUp       = FontAwesomeIcon(faFileArrowUp)
```

Comment:
Why are these `inline def`?

Reply:
In the hopes this will result in smaller code. It's one less level of indirection and most icons are referenced only once.